frontend/public/components/namespace.jsx

```diff
@@ -296,7 +296,7 @@ class NamespaceBarDropdowns_ extends React.Component {
       },
     ];
 
-    return <div className="co-namespace-bar__dropdowns">
+    return <div className="co-namespace-bar__dropdowns pf-l-flex pf-m-justify-content-space-between pf-m-align-items-center">
```

Comment:
No flexbox utility classes please :)

We went down this road in the 3.x console and eventually removed them all. They have a lot of drawbacks.

1. This is basically the same as inline styles, which is considered a bad practice.
2. They don't play well with media queries.
3. If we ever need a flexbox style that doens't have a utility, it's a lot of work to convert this to CSS.

Let's replace this with proper CSS definitions that follow BEM.

Reply:
+1 - i can see how this would be problematic if utility classes were reused in several places and then expected to change in only a few. Will refactor this to use custom classes that use BEM for now. PF4 may introduce more specific layout classes for namespace bar in the future.